domain-model/src/main/java/uk/gov/hmcts/cmc/domain/models/response/ResponseType.java

```diff
@@ -0,0 +1,19 @@
+package uk.gov.hmcts.cmc.domain.models.response;
+
+public enum ResponseType {
+
+    FULL_DEFENCE("reject all of the claim"),
```

Comment:
what do you need this description for?

Reply:
not required atm, but keeping it same as front end for understanding.

Reply:
if not required then please remove

Reply:
done